test/e2e/components/vl-rich-data-table.js

```diff
@@ -24,6 +24,16 @@ class VlRichDataTable extends VlElement {
         return parent.click();
     }
 
+    async getFilterCloseButton() {
+        const button = await this.shadowRoot.findElement(By.css('#close-filter-button'));
+        return new VlButton(this.driver, button);
+    }
+
+    async getFilterToggleButton() {
+        const button = await this.shadowRoot.findElement(By.css('#toggle-filter-button'));
+        return new VlButton(this.driver, button);
+    }
```

Comment:
Ik denk dat we hier liever `closeFilter` en `toggleFilter` hebben dan de interne page objecten te exposen. Klopt hé @pascalriquier?

Reply:
dat lijkt mij beter geencapsuleerd ja